pkg/planner/core/explain.go

```diff
@@ -269,38 +269,55 @@ func (p *PhysicalTableScan) OperatorInfo(normalized bool) string {
 			buffer.WriteString(runtimeFilter.ExplainInfo(false))
 		}
 	}
+
+	annIndexes := make([]string, 0, len(p.UsedColumnarIndexes))
+	invertedIndexes := make([]string, 0, len(p.UsedColumnarIndexes))
 	for _, idx := range p.UsedColumnarIndexes {
-		if idx != nil && idx.QueryInfo.IndexType == tipb.ColumnarIndexType_TypeVector && idx.QueryInfo != nil {
-			buffer.WriteString(", annIndex:")
-			buffer.WriteString(idx.QueryInfo.GetAnnQueryInfo().GetDistanceMetric().String())
-			buffer.WriteString("(")
-			buffer.WriteString(idx.QueryInfo.GetAnnQueryInfo().GetColumnName())
-			buffer.WriteString("..")
+		if idx == nil {
+			continue
+		}
+		if idx.QueryInfo.IndexType == tipb.ColumnarIndexType_TypeVector && idx.QueryInfo != nil {
+			var annIndexBuffer strings.Builder
```

Comment:
This builder should be preallocated and avoid resizing.

ref https://github.com/pingcap/tidb/pull/58285

Reply:
done